examples/src/sysio-dir.c

```diff
@@ -336,7 +336,7 @@ int main(int argc, char **argv)
     MPI_Barrier(MPI_COMM_WORLD);
 
 out_unmount:
-    if (!standard && unmount && rank == 0)
+    if (!standard)
```

Comment:
Did you intend to delete the unmount test here?

Reply:
@adammoody nope :-) Fixed.